warehouse/query-core/src/main/java/datawave/query/transformer/GroupingTransform.java

```diff
@@ -569,4 +591,8 @@ public int hashCode() {
             return hcb.toHashCode();
         }
     }
+    
```

Comment:
Remove this please

Reply:
Done.

Reply:
Please put the hash code function back and remove the commented out lines (now lines 588 to 590). Apologies for not being precise

Reply:
Done in last commit.